src/main/jbake/content/blog/2016/08/31/spark-metric-analysis.adoc

```diff
@@ -0,0 +1,141 @@
+= Metric Data Analysis using the Apache Spark
+Jiri Kremser
+2016-8-31
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, spark, metrics, cassandra
+
+The goal of this blog post is to show how to easily connect link:https://spark.apache.org/[Apache Spark] to the
+Cassandra with the metric data and do some data transformations/analysis.
+
```

Comment:
Agreed with Heiko, a little bit more info into what you will really do here, would be more appealing to read the rest of the article. When I read "do some data transformations/analysis", I don't even have an idea what I would want to do, give me some idea, like the memory correlation (and what that means)